extensions-core/google-extensions/src/test/java/org/apache/druid/data/input/google/GoogleCloudStorageInputSourceTest.java

```diff
@@ -66,6 +66,7 @@
 
 public class GoogleCloudStorageInputSourceTest extends InitializedNullHandlingTest
 {
+  private static final long EXPECTED_MAX_LISTING_LENGTH = 1024L;
```

Comment:
nit: this should probably just use `GoogleCloudStorageInputSource.MAX_LISTING_LENGTH` directly in the event it ever changes